src/components/popover/popover.module.scss

```diff
@@ -0,0 +1,44 @@
+@import 'src/assets/styles/variables/breakpoints';
+@import 'src/assets/styles/variables/typography';
+@import 'src/assets/styles/mixins/font-scale';
+
+$Z-INDEX-POPUP: 10;
+
+.popover-wrapper {
+  display: block;
+  width: fit-content;
+  height: fit-content;
+  cursor: pointer;
+}
+
+.popover {
+  display: block;
+  position: absolute;
+  z-index: $Z-INDEX-POPUP;
+  font-family: var(--rp-ui-base-font-family);
+  background-color: var(--rp-ui-color-bg);
+  border-radius: 8px;
+  box-shadow: var(--rp-ui-base-shadow-secondary);
+  padding: 16px;
+  min-height: 52px;
+  min-width: 160px;
+  max-width: fit-content;
+  box-sizing: border-box;
+  outline: 0;
+  width: max-content;
+  font-size: 11px;
+  line-height: 20px;
```

Comment:
I see you imported `font-scale`, but don't use it.
The idea is to follow common design rules in terms of typography and use strictly defined font sizes.
So please use the `font-scale()` mixin here as it is build based on the [Typography from DS](https://www.figma.com/design/gjYQPbeyf4YsH3wZiVKoaj/%F0%9F%9B%A0-RP-DS-6?node-id=59-894&t=7AuFqgrO5a8UbzsB-0). In case there are any inconsistencies, discuss with UX team and adjust if needed.